slack-base/src/main/java/com/hubspot/slack/client/methods/params/reactions/ReactionsAddParamsIF.java

```diff
@@ -0,0 +1,23 @@
+package com.hubspot.slack.client.methods.params.reactions;
+
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ReactionsAddParamsIF {
+ String getName();
+ /**
+  * One of file, file_comment, or the combination of channel and timestamp must be specified.
```

Comment:
Can we specify this via a [check](https://github.com/HubSpot/hubspot-immutables#validation)? I'm generally a fan of expressing restrictions through code rather than comments.

Reply:
my bad. I meant to add the check. I didn't know we had a convenience method for it though. Included a few tests too